.editorconfig

```diff
@@ -143,6 +143,9 @@ dotnet_diagnostic.CA2100.severity = silent
 # CA1416: Validate platform compatibility
 dotnet_diagnostic.CA1416.severity = silent
 
+# CA1310: Specify StringComparison for correctness
+dotnet_diagnostic.CA1310.severity = error
```

Comment:
As you can see from the build results, we should consider putting this in a .editorconfig file in various subfolders to only apply it to libraries that require it.

YES: MDS, MSS, MDS AKV Provider
MAYBE: Test utility libraries
NEEDS EVAL: MDS Test libraries (I'm guessing a lot of places in test code should be specifying the option, but not sure if all require it, so we might not want to impose this rule on tests.)

Reply:
that is what I was looking for, but nothing on the .editorconfig subfolders helped me. so I went through all the tests and noticed there are almost 17 classes that needs to be converted to ordinal comparison. There were couple of places that ignore case were required. I have changed them all. If you still think that I should go with the editorconfig setup. I will revert these changes and will go with that pattern.